config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -1971,7 +1971,7 @@ susy.quest.db.796249166.desc=- §6Lever Module§r: Works as a simple lever, both
 susy.quest.db.797571327.title=ETERNAL SUFFERING
 susy.quest.db.797571327.desc=The §cShell Higher Olefin Process§r is a method to produce linear alpha olefins. The catalyst is the main hassle in the process, and requires multitudinous steps to synthesize. Good luck, you will need it.%n%nThese linear alpha olefins are essential for producing extraction agents for ore processing in the EV age, and are also useful for producing lubricant base oils and additives.
 susy.quest.db.802655963.title=Combustion Generators
-susy.quest.db.802655963.desc=Unlocked in EV, these combustion generators are a compact way to produce power from refined fuels without the need for and oxidant and a large multiblock! These are rather expensive and require 2 iridium each.
+susy.quest.db.802655963.desc=Unlocked in EV, these combustion generators are a compact way to produce power from refined fuels without the need for an oxidant and a large multiblock! These are rather expensive and require 2 iridium each.
```

Comment:
this is actually inaccurate rn, the LV and MV combustion generators are unlocked in MV using 2 palladium and the HV in EV using 2 iridium